internal/usecase/article.go

```diff
@@ -0,0 +1,17 @@
+package usecase
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/PichuChen/go-bbs"
+)
+
+func (usecase *usecase) GetPopularArticles(ctx context.Context) ([]bbs.ArticleRecord, error) {
```

Comment:
```suggestion
// GetPopularArticles returns ... .... by .... ... 
func (usecase *usecase) GetPopularArticles(ctx context.Context) ([]bbs.ArticleRecord, error) {
```

需要補上註解說明目前usecase 是按照什麼回應人氣文章的，已讀數量？推文數量？

根據 PTT APP 那邊開發會議目前的大方向應該會以「留言數量」(推+噓+箭頭) 作為人氣文章的排序依序

不過目前沒有這項資訊的話，應該也可以寫TBD或是另外開ISSUE討論

Reply:
我看文件上有 `comment count` 說明是留言數量，不分推噓文，我先假設是照這欄排序好了
然後資料是否要 top N 之後再開 issue 討論嗎？

Reply:
如何排序可以回應或者是開ISSUE都可以，這邊Review上面只是針對沒有註解的這件事提出意見而已。